cfme/configure/access_control/__init__.py

```diff
@@ -766,44 +693,6 @@ def delete(self, cancel=True):
             assert view.is_displayed, (
                 "Access Control Group {} Detail View is not displayed".format(self.description))
 
-    def edit_tags(self, tag, value):
```

Comment:
**QUESTION:** This is more for me to learn, but are these deleted because their functionality is part of being a Taggable?

Reply:
@jamesooden Correct. `Taggable` mixin provides it.

Reply:
yes, now we can use it from Taggable class